spec/models/agents/freme_filter_agent_spec.rb

```diff
@@ -58,12 +58,23 @@
 
   describe "#receive" do
     before(:each) do
-      @event = Event.new(payload: {data: "Hello from Huginn"})
+      @event = Event.new(payload: {data:
+         <<-END
```

Comment:
It's a good idea to provide the agent with the correct payload format, but technically not needed because we do not run real HTTP requests